### PR TITLE
feat:adds the rules page

### PR DESCRIPTION
**Pull request by FarhanMobashir**
-adds the rule page with mobilw responsivenens

### CODE REVIEW

**Review by hinanshisuthar — APPROVED** [review]
Looks good to me. Good use of classes. Clean code practices followed. Can be merged!

File: styles/rules.css [hunk 1]
```diff
@@ -1,2 +1,15 @@
...
+}
+
+.rules-container {
+  border: 1px solid var(--black-100);
+  border-radius: var(--radius-sm);
+  padding: 2% 4%;
+  margin-top: var(--sp-lg);
+}
+
+@media (max-width: 600px) {
```
Comment (RakshithBhat03) [inline styles/rules.css:14]: Media query is empty. You can remove it if not needed

**Review by RakshithBhat03 — APPROVED** [review]
Everything looks good. The code is clean and structured well.

**Review by GuptaEesh — APPROVED** [review]
Code looks clean and variables naming is perfect.

**Pull request merged**